alibi_detect/saving/loading.py

```diff
@@ -140,6 +139,14 @@ def _load_detector_config(filepath: Union[str, os.PathLike]) -> ConfigurableDete
     logger.info('Instantiating detector.')
     detector = _init_detector(cfg)
     logger.info('Finished loading detector.')
+
+    # Load state if it exists (and detector supports it)
+    # TODO - this will be removed in follow-up offline state PR, as loading to be moved to __init__ (w/ state_dir kwarg)
+    if isinstance(detector, StatefulDetector):
```

Comment:
Nitpick but perhaps the above logging info could reflect whether it has finished loading a stateless or stateful detector, perhaps mentioning the timestep in the latter case?

Reply:
Nice idea!

Reply:
Done in 9e844105587d850ce9e3325073b60b3c5dcf95d5